e2e/cypress/integration/channel_sidebar/custom_categories_spec.js

```diff
@@ -129,13 +125,20 @@ function createCategoryFromSidebarMenu() {
     // # Start with a new category
     const categoryName = `category-${getRandomId()}`;
 
-    // # Click on the siebar menu dropdown
-    cy.get('.AddChannelDropdown_dropdownButton').click();
+    // # Click on the sidebar menu dropdown
+    cy.findByLabelText('Add Channel Dropdown').click();
 
     // # Click on create category link
-    cy.get('#createCategory').click();
+    cy.findByText('Create New Category').should('be.visible').click();
+
+    // # Verify that Create Category modal has shown up.
+    // # Wait for a while until the modal has fully loaded, especially during first-time access.
+    cy.get('#editCategoryModal').should('be.visible').wait(TIMEOUTS.HALF_SEC).within(() => {
```

Comment:
@josephbaylon Thanks for testing. Looks like the issue is here, especially during first-time access to the modal. Please try again and see if it works for you.

Also, while here, I've fixed others by:
- using `cy.findByLabelText` when targeting with `aria-label`,
- capitalize comments for consistency
- fixed some typo errors